Python/cp/LetterCount.py

```diff
@@ -0,0 +1,31 @@
+"""
+The program uses dictionary in python
+to count the number of occurence a letter 
+makes in a word
+
+"""
+word=input("enter a word ").lower() #Converting to lowercase for the ASCII code to be same
+  
+
+dict={}
+
+for c in word:
```

Comment:
It would be better if you can make separate functions for code logic and driver code.